rdsAutoStopLambdaWorker.py

```diff
@@ -0,0 +1,37 @@
+import json
```

Comment:
Not sure why this file is here, if you want to have a 'Copy' for testing that's OK, lets put it in a folder src/xxx.py and add headers for the license and comment these are for testing and not used in the deployment


code in yaml is what this project deploys

Reply:
addressed